Assets/AirConsole/scripts/AirConsole.cs

```diff
@@ -1930,6 +1939,26 @@ public class NotReadyException: SystemException {
 #endif
 
 #endregion
+		
+#if UNITY_ANDROID || UNITY_EDITOR
+		/// <summary>
+		/// Enables player silencing
+		/// </summary>
+		private void EnablePlayerSilencing () {
+			if (!IsAirConsoleUnityPluginReady ()) {
+				throw new NotReadyException ();
```

Comment:
Is it normal that some functions are called with a space before the brackets and some don't? Is there no C# formatting guideline?

Reply:
There are but not for spaces before squirmy brackets as there are different schools of thoughts on this in C languages. Same for opening squirmy on same line or not and else if 

from what I’ve seen there are certain official guidelines that were not followed (JavaScript casing and some other things, non C# file layout) while others were not consistently applied.

I will, if I have some headspace, see to configure and include Rider formatting that applies those things that don’t break API consistently on save.
Just takes time as C# has some 2xx formatting rules + Unity specificas (which only Riders engine even understands) and about 40 - 50 need to be enable/disabled, applied to the project and get verified.